<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog1</class>
 <widget class="QDialog" name="Dialog1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="0">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>50</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_up">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">position: absolute;
width: 290px;
height: 30px;
left: 150px;
top:25px;


font-style: normal;
font-weight: 600;
font-size: 48px;
line-height: 59px;
text-align: center;

color: #2C2C2C;</string>
       </property>
       <property name="text">
        <string>Конвертор файлов</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QFrame" name="frame">
       <property name="styleSheet">
        <string notr="true">position: absolute;
width: 1117px;
height: 401px;
left: 22px;
top: 153px;

background: #D9D9D9;
border-radius: 25px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>96</number>
          </property>
          <property name="leftMargin">
           <number>36</number>
          </property>
          <property name="topMargin">
           <number>98</number>
          </property>
          <property name="rightMargin">
           <number>36</number>
          </property>
          <property name="bottomMargin">
           <number>189</number>
          </property>
          <item>
           <widget class="QPushButton" name="exel_exel_btn">
            <property name="styleSheet">
             <string notr="true">position: absolute;
width: 282px;
height: 114px;
left: 58px;
top: 251px;

background: #F23026;
border-radius: 25px;

font-weight: 700;
font-size: 31px;
line-height: 38px;
text-align: center;

color: #FFFFFF;</string>
            </property>
            <property name="text">
             <string>exel -&gt; exel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exel_word_btn">
            <property name="styleSheet">
             <string notr="true">position: absolute;
width: 282px;
height: 114px;
left: 58px;
top: 251px;

background: #F23026;
border-radius: 25px;


font-weight: 700;
font-size: 31px;
line-height: 38px;
text-align: center;

color: #FFFFFF;</string>
            </property>
            <property name="text">
             <string>word -&gt; exel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exel_json_btn">
            <property name="styleSheet">
             <string notr="true">position: absolute;
width: 282px;
height: 114px;
left: 58px;
top: 251px;

background: #F23026;
border-radius: 25px;


font-weight: 700;
font-size: 31px;
line-height: 38px;
text-align: center;

color: #FFFFFF;</string>
            </property>
            <property name="text">
             <string>exel -&gt; json</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_down">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">
position: absolute;
width: 486px;
height: 37px;
left: 337px;
top: 450px;

font-weight: 400;
font-size: 27px;
line-height: 32px;
text-align: center;

color: #000000;</string>
          </property>
          <property name="text">
           <string>выберите формат</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
